### PR TITLE
Update aws-alb-aad-ocid-auth.md

### PR DESCRIPTION
**Pull request by cirpo**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: contrib/docs/tutorials/aws-alb-aad-ocid-auth.md [hunk 1]
```diff
@@ -8,24 +8,24 @@ It is assumed an ALB is already serving traffic in front of a Backstage instance
 
 ## Infrastructure setup
 
-### AAD App
+### AAD (Azure Active Directory) App
```
Comment (crivetechie) [inline contrib/docs/tutorials/aws-alb-aad-ocid-auth.md]: I've already introduced AAD earlier, should I repeat?

File: contrib/docs/tutorials/aws-alb-aad-ocid-auth.md [hunk 2]
```diff
@@ -8,24 +8,24 @@ It is assumed an ALB is already serving traffic in front of a Backstage instance
...
+Instead of `localhost` addresses, use the following values.
 
 - Identifier URI: `https://backstage.yourdomain.com`
 - Redirect URI: `https://backstage.yourdomain.com/oauth2/idpresponse`
 
-The values of `Application (client) Id`, `Directory (tenant) ID` and `client secret` will be used while configuring the ALB.
+`Application (client) Id`, `Directory (tenant) ID` and `client secret`values will be used while configuring the ALB.
 
-### ALB
+### AWS ALB (Applicaton Load Balancer)
```
Comment (crivetechie) [inline contrib/docs/tutorials/aws-alb-aad-ocid-auth.md]: I've already introduced ALB earlier, do you recommend repeating?

File: contrib/docs/tutorials/aws-alb-aad-ocid-auth.md [hunk 3]
```diff
@@ -41,13 +41,14 @@ Use the following advanced settings:
...
 
-Once you've saved the action you should see an authentication flow be triggered against AAD when visiting Backstage address at `https://backstage.yourdomain.com`, the flow will not complete successfully as Backstage app isn't yet configured properly.
+Once you've saved the action, you should see an authentication flow be triggered against AAD when visiting Backstage address at `https://backstage.yourdomain.com`. The flow will not complete successfully as Backstage app isn't yet configured properly.
 
 ## Backstage changes
 
 ### Frontend
 
-The Backstage App requires a SingInPage when authentication is required. When using ALB authentication Backstage will only be loaded once the user has successfully authenticated; we won't need to display a SignIn page, however we will need to create a dummy SignIn component that can refresh the token.
+The Backstage App requires a SingInPage when authentication is required.
```
Comment (crivetechie) [inline contrib/docs/tutorials/aws-alb-aad-ocid-auth.md]: using 'requires' and 'required'... it feels like we can use a better wording here

File: contrib/docs/tutorials/aws-alb-aad-ocid-auth.md (hunk 1 above)
Comment (cirpo) [reply contrib/docs/tutorials/aws-alb-aad-ocid-auth.md]: no idea why I missed it

Comment (cirpo) [inline contrib/docs/tutorials/aws-alb-aad-ocid-auth.md]:
```suggestion
### AAD App
```

File: contrib/docs/tutorials/aws-alb-aad-ocid-auth.md (hunk 2 above)
Comment (cirpo) [inline contrib/docs/tutorials/aws-alb-aad-ocid-auth.md]:
```suggestion
### ALB
```

File: contrib/docs/tutorials/aws-alb-aad-ocid-auth.md (hunk 3 above)
Comment (cirpo) [inline contrib/docs/tutorials/aws-alb-aad-ocid-auth.md]:
```suggestion
The Backstage App needs a SingInPage when authentication is required. 
```

**Review by crivetechie — APPROVED** [review]

**Pull request merged**